docker/opensource.md

```diff
@@ -86,6 +86,23 @@ $ docker run -it --rm --workdir /opt -v$(pwd):/opt ubuntu:22.10 bash
 # composer install
 # phpunit
 ```
+## capella
 
+* [capella](https://github.com/AlexSpradling/Capella)
+* I have added `CONTRIBUTING.rst`
+
+```
+$ git clone https://github.com/AlexSpradling/Capella.git
+$ cd capella
```

Comment:
the name of the folder is with a capital C
this cd will only work on Windows which is case insensitive